redis/src/cluster.rs

```diff
@@ -410,7 +410,7 @@ where
         let params = self.cluster_params.clone();
         let info = get_connection_info(node, params)?;
 
-        let mut conn = C::connect(info, None)?;
```

Comment:
If I'm reading this right, we effectively don't use connection timeout settings in this client? This seems like a potentially impactful to merit a more detailed commit message/possibly a separate PR?

Reply:
I'll open this as a separate PR, then.